js/reference.js

```diff
@@ -2832,7 +2832,27 @@ var ERMrest = (function(module) {
                 return null;
             }
 
-        }
+        },
+
+        /**
+         * @desc
+         * This function takes the current Tuple (this) and creates a shallow copy of it while de-referencing
+         * the _data attribute. This way _data can be modified in chaise without changing the originating Tuple
+         * @returns {ERMrest.Tuple} a shallow copy of _this_ tuple with it's _data de-referenced
+         */
+         deepCopyData: function() {
+             var newTuple = Object.create(Tuple.prototype);
+             module._clone(newTuple, this);
```

Comment:
I now this isn't part of your diff but `_clone` should be renamed to `_shallowCopy` because its really unclear that 'clone' is supposed to mean shallow copy.

If its a quick and easy change, do it, but if its not easy don't.